### PR TITLE
[20529] Add skip_existing option to vcs_import action

### PR DESCRIPTION
**Pull request by EduPonz**
## Description

This PR adds a `skip_existing` boolean option to the `vcs_import` action.

## Contributor Checklist
- [x] Commit messages follow the company guidelines. 
- [x] Changes are API compatible. 
- [x] New features have been added to the `versions.md` and `README.md` files (if applicable).

## Reviewer Checklist
- [x] The title and description correctly express the PR's purpose.
- [x] The Contributor checklist is correctly filled.

### CODE REVIEW

**Review by JesusPoderoso — APPROVED** [review]
LGTM

File: ubuntu/vcs_import/action.yml [hunk 1]
```diff
@@ -26,7 +31,13 @@ runs:
 
         mkdir -p ${{ inputs.destination_workspace }}
 
-        vcs import ${{ inputs.destination_workspace }} < ${{ inputs.vcs_repos_file }}
+        SKIP_EXISTING=""
+        if [[ "${{ inputs.skip_existing }}" == "true" ]]
+        then
+          SKIP_EXISTING="--skip_existing"
```
Comment (JesusPoderoso) [inline ubuntu/vcs_import/action.yml:37]:
```suggestion
          SKIP_EXISTING="--skip-existing"
```

File: windows/vcs_import/action.yml [hunk 2]
```diff
@@ -24,6 +29,11 @@ runs:
 
         New-Item -ItemType Directory -Path ${{ inputs.destination_workspace }} -Force
 
-        vcs import ${{ inputs.destination_workspace }} --input ${{ inputs.vcs_repos_file }}
+        $SKIP_EXISTING = $null
+        if ("${{ inputs.skip_existing }}" -eq "true") {
+          $SKIP_EXISTING = "--skip_existing"
```
Comment (JesusPoderoso) [inline windows/vcs_import/action.yml:34]:
```suggestion
          $SKIP_EXISTING = "--skip-existing"
```

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]
NIT according [this](https://github.com/eProsima/Fast-DDS/actions/runs/8062697354/job/22022975664?pr=4456)

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Comment by EduPonz** [comment]
Merging this, it was tested successfully [here](https://github.com/eProsima/Fast-DDS/actions/runs/8062697354/job/22039143601?pr=4456#step:8:64)

**Pull request merged**